### DISCUSSION TITLE
Document showing differently for cognito identity pool

### DISCUSSION THREAD

**Discussion by linuxananth1976** [cloud-custodian/cloud-custodian#8096] [General]
Both were referring to same aws resource cognito identity pool but displays different options.

https://cloudcustodian.io/docs/aws/resources/identity-pool.html -  shows more options especially tag and mark-for-op

https://cloudcustodian.io/docs/aws/resources/cognito-identity.html#aws-identity-pool - not update i believe

Please update the latest. also one more request user-pool won't support for tags.

**Comment by ajkerrigan** [comment]
How are you getting to https://cloudcustodian.io/docs/aws/resources/cognito-identity.html#aws-identity-pool ? I don't see that page in the current docs site or any pages that reference it (and as a check, [that path on GitHub Pages](https://cloud-custodian.github.io/cloud-custodian/docs/aws/resources/cognito-identity.html#aws-identity-pool) goes nowhere), so wondering where you're seeing the link. The closest current relative would be https://cloudcustodian.io/docs/aws/resources/group-cognito-identity.html .

I'm guessing that older URL is an orphaned page we should remove or redirect from, but that's not something we can handle from the custodian repo itself.

**Comment by linuxananth1976** [comment]
Yes its somewhere linked, from document side it works fine. Thanks for the update
